guava-compatibility-agent/src/main/java/com/palantir/guavacompat/agent/Agent.java

```diff
@@ -74,10 +76,12 @@ public static void premain(String _args, Instrumentation instrumentation) {
         agentBuilder = agentBuilder
                 .type(ElementMatchers.named(FUTURES))
                 .transform((builder, _type, _classLoader, _module, _protection) -> {
-                    TypeDescription.Generic inputTypeVariable =
+                    TypeDescription.Generic typeVarI =
                             TypeDescription.Generic.Builder.typeVariable("I").build();
-                    TypeDescription.Generic outputTypeVariable =
+                    TypeDescription.Generic typeVarO =
                             TypeDescription.Generic.Builder.typeVariable("O").build();
+                    TypeDescription.Generic typeVarV =
```

Comment:
Should we use this in this code below?:

```@Override
public TypeList.Generic getTypeVariables() {
  return new TypeList.Generic.Explicit(
    TypeDescription.Generic.Builder.typeVariable("V")
      .build());
}
```

Reply:
I think you're right, good call -- The `TypeList` can wrap the typeVariables we've defined above -- I'll fix it up!